en/setup/bibtexkeypatterns.md

```diff
@@ -9,6 +9,7 @@ The pattern used in the auto generation of BibTeX labels can be set for each of
 ## Key patterns
 
 The key pattern can contain any text you wish, in addition to field markers that indicate that a specific field of the entry should be inserted at that position of the key. A field marker generally consists of the field name enclosed in square braces, e.g. **\[volume\]**. If the field is undefined in an entry at the time of key generation, no text will be inserted by the field marker.
+**Note:** if you are using JabRef 5, in order for your changes to be retained, you must hit "enter" on your keyboard before clicking on the "Save" button.
```

Comment:
The help is always for the latest stable release, so the introduction "if you are using JabRef 5" is not needed.

Reply:
Done